test/Rules.js

```diff
@@ -1,252 +1,336 @@
 var assert = require("assert");
 var Rules = require("../lib/Rules");
 
+describe("Rules", function () {
```

Comment:
All your suggestions are good ones, and good optimizations.

I will be sure to implement as requested and keep you posted.

Thanks!

On Mon, Aug 3, 2015 at 4:31 PM, Will Stern <notifications@github.com> wrote:

> In test/Rules.js
> <https://github.com/willrstern/rules-runner/pull/8#discussion_r36096946>:
>
> > @@ -1,252 +1,336 @@
> >  var assert = require("assert");
> >  var Rules = require("../lib/Rules");
> >
> > +describe("Rules", function () {
>
> Since this project's 2-spaced instead of 4, would you be up for changing
> this to 2-spaced indents and re-pushing, so I can tell what's changed?
>
> I really should ad an .editorconfig in this project. [image: :frowning:]
>
> —
> Reply to this email directly or view it on GitHub
> <https://github.com/willrstern/rules-runner/pull/8/files#r36096946>.
>